pages/event-types/index.tsx

```diff
@@ -701,13 +700,10 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
     } as const;
   }
 
-  if (!user.completedOnboarding && dayjs(user.createdDate).isAfter(ONBOARDING_INTRODUCED_AT)) {
-    return {
-      redirect: {
-        permanent: false,
-        destination: "/getting-started",
-      },
-    } as const;
+  if (
+    shouldShowOnboarding({ completedOnboarding: user.completedOnboarding, createdDate: user.createdDate })
```

Comment:
think this should work.

```suggestion
    shouldShowOnboarding(user)
```